src/logic/proposals/encoding/encoding.test.ts

```diff
@@ -20,8 +21,8 @@ const newFormData = {
   type: 'primary' as const,
 };
 const api3Agent = {
-  primary: '0xd001cbCb278dDb424964e894a98d26bBdd4A8679',
-  secondary: '0xb46863F7014D6F100F67Af778d660647f9Fc96ff',
+  primary: '0xd9f80bdb37e6bad114d747e60ce6d2aaf26704ae',
+  secondary: '0x556ecbb0311d350491ba0ec7e019c354d7723ce0',
```

Comment:
I used a proposal from mainnet in these tests, so I updated the agent addresses here (to be the ones from mainnet) so that the re-encoding matches the original evm script